langwatch/prisma/migrations/20260715000100_create_webhook_delivery/migration.sql

```
@@ -0,0 +1,44 @@
+    "outcome" "WebhookDeliveryOutcome" NOT NULL,
+    "firedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
+    CONSTRAINT "WebhookDelivery_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE INDEX "WebhookDelivery_projectId_triggerId_dispatchId_idx" ON "WebhookDelivery"("projectId", "triggerId", "dispatchId");
+
+-- CreateIndex
+CREATE INDEX "WebhookDelivery_projectId_triggerId_firedAt_idx" ON "WebhookDelivery"("projectId", "triggerId", "firedAt");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)